Android/README.md

```diff
@@ -80,18 +80,29 @@ call. For example, if you want a pydebug build that also caches the results from
 
 ## Testing
 
-To run the Python test suite on Android:
-
-* Install Android Studio, if you don't already have it.
-* Follow the instructions in the previous section to build all supported
-  architectures.
-* Run `./android.py setup-testbed` to download the Gradle wrapper.
-* Open the `testbed` directory in Android Studio.
-* In the *Device Manager* dock, connect a device or start an emulator.
-  Then select it from the drop-down list in the toolbar.
-* Click the "Run" button in the toolbar.
-* The testbed app displays nothing on screen while running. To see its output,
-  open the [Logcat window](https://developer.android.com/studio/debug/logcat).
-
-To run specific tests, or pass any other arguments to the test suite, edit the
-command line in testbed/app/src/main/python/main.py.
+Before running the test suite, follow the instructions in the previous section
+to build for all supported architectures.
+
+The test suite can be run in two modes:
+
+* In `--connected` mode, it runs on a device or emulator you have already
+  connected to the build machine. For example:
+
+  ```sh
+  ./android.py test --connected emulator-5554
+  ```
+
+* In `--managed` mode, it uses a temporary emulator defined in the
+  `managedDevices` section of testbed/app/build.gradle.kts. This mode is slower,
+  but more reproducible. For example:
+
+  ```sh
+  ./android.py test --managed targetSdk
+  ```
+
+By default, the only messages the script will show are Python's own stdout and
+stderr. Add the `-v` option to also show Gradle output, and non-Python logcat
+messages.
+
+Any other arguments on the `android.py test` command line will be passed through
+to `python -m test`. Use `--` to separate them from android.py's own options.
```

Comment:
If I'm reading this right, the args default to `-W -uall`; this is definitely a good set of default arguments, but it might be worth mentioning that they will be applied automatically.

It would also be worth mentioning that `-j`/`--multiprocess` isn't a valid option (I presume this is also true of Android?). Better still would be to catch that argument and strip it from the active arguments.

Reply:
> the args default to `-W -uall`

It doesn't look like there's any way to switch from `-W` to `-v` mode by appending arguments, so for simplicity I've now changed it to default to no arguments at all. That will make the buildbot worker script easier to understand anyway, because it'll correspond more directly to `python -m test` which the other platforms use.

> It would also be worth mentioning that `-j`/`--multiprocess` isn't a valid option

I've added this to the README.